From 2bccb5f0a14036ac853997a741f32f94a5107405 Mon Sep 17 00:00:00 2001
From: Liming Wang <liming.wang@windriver.com>
Date: Tue, 4 Dec 2012 10:01:24 +0800
Subject: [PATCH 26/26] xilinx_zynq: Comment the second ehci controller

Qemu now doesn't support two ehci controller well. The usb device will always
be attached to the second ehci controller, which we don't need. For linux,
we only support the first ehci controller, so reserve the firt one to model.

Signed-off-by: Liming Wang <liming.wang@windriver.com>
---
 hw/xilinx_zynq.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hw/xilinx_zynq.c b/hw/xilinx_zynq.c
index 2a188b7..bfc2929 100644
--- a/hw/xilinx_zynq.c
+++ b/hw/xilinx_zynq.c
@@ -202,7 +202,7 @@ static void zynq_init(QEMUMachineInitArgs *args)
     zynq_init_spi_flashes(0xE000D000, pic[51-IRQ_OFFSET], true);
 
     sysbus_create_simple("xlnx,ps7-usb", 0xE0002000, pic[53-IRQ_OFFSET]);
-    sysbus_create_simple("xlnx,ps7-usb", 0xE0003000, pic[75-IRQ_OFFSET]);
+    //sysbus_create_simple("xlnx,ps7-usb", 0xE0003000, pic[75-IRQ_OFFSET]);
 
     sysbus_create_simple("cadence_uart", 0xE0000000, pic[59-IRQ_OFFSET]);
     sysbus_create_simple("cadence_uart", 0xE0001000, pic[82-IRQ_OFFSET]);
-- 
1.7.9.5

